### PR TITLE
admin_gvpn

### PR DESCRIPTION
**Pull request by saumitraaditya**
1. contains new code for MuC feature and utility scripts to create/configure and manage users.
2. renamed variable "self.state" to "self.ipop_state" in ipoplib and svpn,gvpn controllers.
3. changed signature of "do_get_state" method in ipoplib,svpn,gvpn.

### CODE REVIEW

**Comment by kyuhojeong** [comment]
For the coding style, we follow PEP 8 rules(https://www.python.org/dev/peps/pep-0008/). So, we should use # for the comments and for the multiple lines of comments instead of ''' (https://www.python.org/dev/peps/pep-0008/#block-comments)

File: src/create_room.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+room_members_by_default : True
+room_allow_private_messages : True
+room_moderated : False
+
+Note: JID used to create the room will be the owner and will have all
+admin rights, same JID should be used when managing users subsequently.
+'''
+
+
+#!/usr/bin/env python
```
Comment (kyuhojeong) [inline src/create_room.py]: bangline should be at the top of the code.

File: src/manageUsers.py [hunk 2]
```diff
@@ -0,0 +1,232 @@
+
+#!/usr/bin/env python
```
Comment (kyuhojeong) [inline src/manageUsers.py]: Again, shebang should be the first line here.

File: src/manageUsers.py [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+'''Randomly generate a IP address, checks if it is unassigned
+and allocate it to a new user.also make a note of it in the database.
+currently generates a address in the range (ip prefix && 255.255.xxx.xxx).
+can be modified to accomdate any other range.'''            
+def allocateIP(prefix,user_jid):
+    parts = prefix.split(".")
+    ip_prefix = parts[0] + "." + parts[1] + "."
+    ip_table = shelve.open('ip_table.db',writeback=True)
+    jid_ip_table = shelve.open('jid_ip_table.db',writeback=True)
+    ip_prefix = ip_prefix + str(random.randint(0,255)) + "." + \
```
Comment (kyuhojeong) [inline src/manageUsers.py]: IP address ending 0 is not used. 
Also IP address ending 255 is reserved for broadcast address. 
So randint range should be 1, 254

**Comment by saumitraaditya** [comment]
code made PEP8 compliant
IPv4 allocation range fixed in manageUsers.py to (1,254)

**Pull request merged**